src/pages/superadmin/tableComponent/index.tsx

```diff
@@ -485,26 +485,28 @@ export const MyTable = ({
                         target="_blank"
                         rel="noopener noreferrer"
                       >
-                        {bounty?.title}
+                        {bounty?.title.length > 50
+                          ? `${bounty?.title.substring(0, 47)}...`
+                          : bounty?.title}
                       </a>
                     </BountyData>
                     <TableData>{created}</TableData>
                     <TableDataCenter>{time_to_pay}</TableDataCenter>
                     <TableDataAlternative>
                       <ImageWithText
-                        text={bounty?.assignee}
+                        text={bounty?.assignee_alias}
                         image={bounty?.assignee_img || defaultPic}
                       />
                     </TableDataAlternative>
                     <TableDataAlternative className="address">
                       <ImageWithText
-                        text={bounty?.owner_pubkey}
-                        image={bounty?.providerImage || defaultPic}
+                        text={bounty?.owner_unique_name}
+                        image={bounty?.owner_img || defaultPic}
                       />
                     </TableDataAlternative>
                     <TableData className="organization">
                       <ImageWithText
-                        text={bounty?.organization}
+                        text={bounty?.organization_name}
```

Comment:
This change is causing the jest test to fail

Reply:
@kevkevinpal yeah changed the mock data as required